src/lib/formValidator.ts

```diff
@@ -19,7 +19,7 @@ export class FormValidator {
     return fields.reduce((validatorResponse, [fieldId, value]) => {
       if (!(fieldId in validatorLookup))
         throw new ReferenceError(
-          `${fieldId} not found in validatorLookup.  Create a new validator key/value pair for this field?`
+          `${fieldId} not found in fieldValidatorLookup.  Create a new validator key/value pair for this field?`
```

Comment:
Just thinking ahead to the upcoming pages and the vessel communications page has 10 inputs that are all optional which would make the case for not having this check in here.  It's a balancing act, but generally I find if the default is to not validate a field then it makes it easier as a developer to not have to specify it.

The trade of being that you need to remember to add a validator but personally I think that's okay as if you're implementing a ticket where a field needs to be validated then it forms part of the acceptance criteria for the ticket so should be specified?

Reply:
@Ninamma Has covered this off in her PR I believe

Reply:
Seen, very elegant way of doing it @Ninamma!  I won't touch on this PR so @Ninamma's solution can merge OK 👍🏻